testutil/contract.go

```diff
@@ -75,7 +75,7 @@ func DeployContractWithFactory(
 	evmosApp *app.Evmos,
 	priv cryptotypes.PrivKey,
 	factoryAddress common.Address,
-	queryClientEvm evm.QueryClient,
+	_ evm.QueryClient,
```

Comment:
we should be able to remove this unused parameter as it's only in the `testutil` package right?